tests/core.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from "chai"
+import * as core from "../src/core"
+
+describe("core translating functionality", () => {
+  it("should detect English language", async () => {
+    const translationContext = await core.checkAndTranslate("i am testing a bot")
+    expect(translationContext.translation.detectedLangCode).to.equal("en")
+    expect(translationContext.translation.detectedLangName).to.equal("English")
+    expect(translationContext.isCorrectLang).to.equal(true)
```

Comment:
How does it know the correct language is English?

Reply:
it used the one in `.env`, which is obviously bad...